tests/tests.toml

```
@@ -4282,6 +4282,10 @@ filename = "multi_error1.f90"
 ast = true # TODO: remove this and only use continue compilation
 continue_compilation = true
 
+[[test]]
+filename = "multiple_objects_args.f90"
+run =true
+
```

```suggestion
pass = "class_constructor"
```
Let's keep a reference of the ASR after the `class_constructor` pass.